Automattic/sensei
# Update courses page navigation

Part of #4529 

### Changes proposed in this Pull Request

* Add the custom menu to the courses page.
* Re-add Order courses page.
* Allow open particular tab of Sensei settings.

### Testing instructions
* Open courses admin page.
* Observe custom navigation.
* Click all the links and make sure the "Courses" menu item is active in the sidebar.
* Try to resize the window to look at changes in the navigation.

### Screenshot / Video
<img width="2005" alt="CleanShot 2022-01-26 at 15 59 26@2x" src="https://user-images.githubusercontent.com/329356/151167491-e4a9c864-8966-4d26-91e3-1f290235a2f9.png">
<img width="935" alt="CleanShot 2022-01-26 at 16 00 09@2x" src="https://user-images.githubusercontent.com/329356/151167502-33ae4b57-2e14-4c53-b6ab-777718def9f9.png">
<img width="520" alt="CleanShot 2022-01-26 at 16 00 26@2x" src="https://user-images.githubusercontent.com/329356/151167508-ef278c19-14a5-4aa2-b8fd-a43b7eb8c825.png">
<img width="376" alt="CleanShot 2022-01-26 at 16 00 46@2x" src="https://user-images.githubusercontent.com/329356/151167518-32b8b24b-d574-42ed-9142-4f950a25c747.png">
